showpass/Showpass-E2E-tests
# TA-37-Verify 'event 1' API payload

**Description:**
This spec will be verifying the 'Test Automation event 1' API payload.
Steps:
- Navigate to [Test Automation event 1 (DO NOT DELETE)](https://beta.showpass.com/ta-event-1/) 
- Open API payload
- Make sure all expected items are still present in the payload

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-37

**How to test:**
Run the following spec and ensure the corresponding test passes.
cypress/e2e/api-requests/events-api.cy.js

**Test execution results:**

<img width="853" alt="TA-37-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/5f5a5368-e811-4847-ba03-ba0fb85077d4">
